rpm/qtdeclarative.spec

```diff
@@ -289,6 +311,17 @@ cp lib/libQt5QmlDevTools.a %{buildroot}/%{_libdir}
 %fdupes %{buildroot}/%{_includedir}
 
 
+# Copy docs
+%if 0%{?qt5_release_build}
+mkdir -p %{buildroot}/%{_docdir}/qt5
+cp -R doc/qtqml %{buildroot}/%{_docdir}/qt5
+cp -R doc/qtqml.qch %{buildroot}/%{_docdir}/qt5
+cp -R doc/qtquick %{buildroot}/%{_docdir}/qt5
+cp -R doc/qtquick.qch %{buildroot}/%{_docdir}/qt5
+%else
```

Comment:
I guess this %else is unnecessary? (I suggested doing it in the build step so you could leave this all unconditional, but if you have the %if here too then you just don't need the %else I suppose)